torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -47,8 +47,7 @@ __attribute__((weak)) int acc_get_device_type() {
 #endif // USE_KINETO
 
 namespace torch {
-namespace autograd {
-namespace profiler {
+namespace autograd::profiler {
```

Comment:
do we need to combine these? I feel the normal convention in pytorch separates each namespace into their own lines

Reply:
yes it fails lint_root with modernize-concat-nested-namespaces

Reply:
We can do whichever you think is more important but I'd rather get rid of all the static errors whenever a diff touches this file